test/scenarios/ynETH.spec.sol

```diff
@@ -315,160 +315,4 @@ contract YnETHScenarioTest8 is IntegrationBaseTest, YnETHScenarioTest3 {
-
-		// There should be a delayedWithdraw on the DelayedWithdrawalRouter
-		IDelayedWithdrawalRouter withdrawalRouter = IDelayedWithdrawalRouter(chainAddresses.eigenlayer.DELAYED_WITHDRAWAL_ROUTER_ADDRESS);
-		IDelayedWithdrawalRouter.DelayedWithdrawal[] memory delayedWithdrawals = withdrawalRouter.getUserDelayedWithdrawals(address(stakingNode));
-		assertEq(delayedWithdrawals.length, 1);
-		assertEq(delayedWithdrawals[0].amount, amount);
-
-		// Because of the delay, the delayedWithdrawal should not be claimable yet
-		IDelayedWithdrawalRouter.DelayedWithdrawal[] memory claimableDelayedWithdrawals = withdrawalRouter.getClaimableUserDelayedWithdrawals(address(stakingNode));
-		assertEq(claimableDelayedWithdrawals.length, 0, "claimableDelayedWithdrawals.length != 0");
-
-		// Move ahead in time to make the delayedWithdrawal claimable
-		vm.roll(block.number + withdrawalRouter.withdrawalDelayBlocks() + 1);
-		IDelayedWithdrawalRouter.DelayedWithdrawal[] memory claimableDelayedWithdrawalsWarp = withdrawalRouter.getClaimableUserDelayedWithdrawals(address(stakingNode));
-		assertEq(claimableDelayedWithdrawalsWarp.length, 1, "claimableDelayedWithdrawalsWarp.length != 1");
-		assertEq(claimableDelayedWithdrawalsWarp[0].amount, amount, "claimableDelayedWithdrawalsWarp[0].amount != 3 ether");
-
-		withdrawalRouter.claimDelayedWithdrawals(address(stakingNode), type(uint256).max);
-
-		return amount;
-	}
-
-	function log_balances (IStakingNode stakingNode) public {
-		emit LogUint("yneth.balance", address(yneth).balance);
-		emit LogUint("stakingNode.balance", address(stakingNode).balance);
-		emit LogUint("consensusReciever.balance", address(consensusLayerReceiver).balance);
-		emit LogUint("executionReciever.balance", address(executionLayerReceiver).balance);
-        emit LogUint("eigenPod.balance", address(IEigenPod(stakingNode.eigenPod())).balance);
-	}
-
-	function runInvariants(address user, uint256 previousTotalDeposited, uint256 previousTotalShares, uint256 userAmount, uint256 userShares) public {
-		
-		uint256 totalDeposited = 0;
-		Vm.Log[] memory logs = vm.getRecordedLogs();
-
-		for (uint i = 0; i < logs.length; i++) {
-			Vm.Log memory log = logs[i];
-			if (log.topics[0] == 
-				keccak256("Deposit(address,address,uint256,uint256,uint256)")) {
-				(uint256 assets,) = abi.decode(log.data, (uint256, uint256));
-				totalDeposited += assets;
-			}
-		}
-		Invariants.totalDepositIntegrity(totalDeposited, previousTotalDeposited, userAmount);
-
-		Invariants.totalAssetsIntegrity(yneth.totalAssets(), previousTotalDeposited, userAmount);
-
-		Invariants.shareMintIntegrity(yneth.totalSupply(), previousTotalShares, userShares);
-		
-		Invariants.userSharesIntegrity(yneth.balanceOf(user), 0, userShares);
-	}
-
-	/*
-
-	*/
-}
-
-// Add this contract definition outside of your existing contract definitions
-contract SelfDestructSender {
-    constructor(address payable _target) payable {
-        selfdestruct(_target);
-    }
-}
-
-
-
+}
```

Comment:
these tests shouldn't be deleted, not sure what was the intention here

Reply:
blunder. added in last commit